prime-angular-frontend/src/app/modules/adjudication/shared/components/notification-email-view/notification-email-view.component.html

```diff
@@ -1,30 +1,170 @@
-                    class="mb-4"
-                    [divider]="false">
+<app-page-subheader>
   <ng-container appPageSubheaderTitle>
-    {{ emailTemplate.templateName }}
+    Email Template Title
   </ng-container>
+  <button *ngIf="Role.MAINTENANCE | inRole"
+          mat-icon-button
+          [matTooltip]="(!titleEditable) ? 'Edit Email Title' : 'Cancel Editing'"
+          (click)="toggleEditTitle(!titleEditable)">
+    <mat-icon>{{ (!titleEditable) ? 'edit' : 'close' }}</mat-icon>
+  </button>
 </app-page-subheader>
-
+<ng-container *ngIf="!titleEditable">
+  {{ emailTemplate?.templateName }}
+</ng-container>
+<ng-container *ngIf="titleEditable">
+  <form [formGroup]="form"
+        (ngSubmit)="saveTitle()">
+    <mat-form-field appearance="fill"
+                    class="w-100">
+      <mat-label>Email Template Title</mat-label>
+      <textarea formControlName="emailTitle"
+                matInput
+                cdkTextareaAutosize
+                #autosize="cdkTextareaAutosize"
+                cdkAutosizeMinRows="1"
+                cdkAutosizeMaxRows="0"></textarea>
+    </mat-form-field>
+    <div class="d-flex justify-content-end">
+      <button mat-flat-button
+              color="primary">
+        Save
+      </button>
+    </div>
+  </form>
+</ng-container>
+<br/>
+<br/>
+<app-page-subheader>
+  <ng-container appPageSubheaderTitle>
+    Description
+  </ng-container>
+  <button *ngIf="Role.MAINTENANCE | inRole"
+          mat-icon-button
+          [matTooltip]="(!descriptionEditable) ? 'Edit Email Description' : 'Cancel Editing'"
+          (click)="toggleEditDescription(!descriptionEditable)">
+    <mat-icon>{{ (!descriptionEditable) ? 'edit' : 'close' }}</mat-icon>
+  </button>
+</app-page-subheader>
+<ng-container *ngIf="!descriptionEditable">
+  {{ emailTemplate?.description }}
+</ng-container>
+<ng-container *ngIf="descriptionEditable">
+  <form [formGroup]="form"
+        (ngSubmit)="saveDescription()">
+    <mat-form-field appearance="fill"
+                    class="w-100">
+      <mat-label>Description</mat-label>
+      <textarea formControlName="description"
+                matInput
+                cdkTextareaAutosize
+                #autosize="cdkTextareaAutosize"
+                cdkAutosizeMinRows="1"
+                cdkAutosizeMaxRows="0"></textarea>
+    </mat-form-field>
+    <div class="d-flex justify-content-end">
+      <button mat-flat-button
+              color="primary">
+        Save
+      </button>
+    </div>
+  </form>
+</ng-container>
+<br/>
+<br/>
+<app-page-subheader>
+  <ng-container appPageSubheaderTitle>
+    Recipient
+  </ng-container>
+  <button *ngIf="Role.MAINTENANCE | inRole"
+          mat-icon-button
+          [matTooltip]="(!recipientEditable) ? 'Edit Email Recipient' : 'Cancel Editing'"
+          (click)="toggleEditRecipient(!recipientEditable)">
+    <mat-icon>{{ (!recipientEditable) ? 'edit' : 'close' }}</mat-icon>
+  </button>
```

Comment:
Should this field be editable?  And if so, should there be some validation?  
![image](https://github.com/bcgov/moh-prime/assets/7586665/94e8a74e-5c4a-424d-a174-984b9aa1e122)

Reply:
This field is for information only. I don't think it is necessary to have a validation.